federation/pkg/federation-controller/daemonset/daemonset_controller_test.go

```diff
@@ -38,27 +38,33 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const (
+	daemonsets string = "daemonsets"
+	clusters   string = "clusters"
+	errorMsgF  string = "expected: %v, actual: %v"
```

Comment:
const printf-style format strings make them less readable in the code where they are used.